include/admin_navigation.js

```diff
@@ -257,14 +305,26 @@ module.exports = function AdminNavigationModule(pb) {
      * @method buildNavigation
      * @returns {Array}
      */
-    function buildNavigation() {
+    function buildNavigation(site) {
         var i;
         var navigation = [];
-        var defaultNavigation = getDefaultNavigation();
-        var additions = getAdditions();
-        var childrenAdditions = getChildrenAdditions();
+        var multiSiteAdditions = getMultiSiteNavigation();
+        var defaultNavigation = getDefaultNavigation(site);
```

Comment:
The calls for multi-stie navigation and default navigation don't always have to be made.  Only make the function call inside of the if/else block that they pertain too.